cpp/core/operators/c2r/ColumnarToRow.h

```diff
@@ -27,7 +28,15 @@ class ColumnarToRowConverter {
 
   virtual ~ColumnarToRowConverter() = default;
 
-  virtual void convert(std::shared_ptr<ColumnarBatch> cb = nullptr) = 0;
+  // We will start conversion from the 'rowId' row of 'cb'. The maximum memory consumption during the grabbing and
+  // swapping process is 'memoryThreshold' bytes. The number of rows successfully converted is stored in the 'numRows_'
+  // variable.
+  virtual void
+  convert(std::shared_ptr<ColumnarBatch> cb = nullptr, int64_t rowId = 0, int64_t memoryThreshold = INT64_MAX) = 0;
```

Comment:
could we configure `memoryThreshold` for ColumnarToRowConverter when initializing it?

Reply:
@zhztheplayer can you fix it?